server.js

```diff
@@ -18,11 +44,37 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
-app.get('/', (req, res) => {
-  res.send('Hello world')
+
+app.get('/', async (req, res) => {
+  const thoughts = await Thought.find().sort({ createdAt: 'desc' }).limit(20).exec()
+  if (thoughts.length > 0) {
+    res.json(thoughts)
+  } else {
+    res.status(400).json({ message: "Fill up this empty space with your happiness" })
```

Comment:
Do you really want to return a 400 if the user just started using your site for the first time?